
number_of_robots: 2
robot1: "turbot"
robot2: "xiroi"
